components/RichText/RichText.tsx

```diff
@@ -97,6 +98,15 @@ const getRenderOptions = (links?: any) => {
                                 </Quote>
                             );
                         }
+
+                        if (entry.__typename === 'YouTubeEmbed') {
+                            return (
+                                <YouTubeEmbed
+                                    url={entry.videoUrl}
+                                    title={entry.videoTitle}
```

Comment:
I assume this `videoTitle` will always be present? I think youtube require it now. Is this coming from the contentful api?

Reply:
I assume that it is from the entry in api.ts below